src/components/layertree/LayerTree.js

```diff
@@ -0,0 +1,389 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Tree, { mutateTree } from '@atlaskit/tree';
+import shortid from 'shortid';
+
+const propTypes = {
+  /**
+   * The object representing a hierachical object to display as a tree.
+   * This property is only used at initialisation time. Use onItemChange if
+   * you want to keep track of the tree status.
+   */
+  tree: PropTypes.shape({
+    rootId: PropTypes.string.isRequired,
+    items: PropTypes.objectOf(
+      PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        parentId: PropTypes.string,
+        children: PropTypes.arrayOf(PropTypes.string).isRequired,
+        hasChildren: PropTypes.bool.isRequired,
+        hasParent: PropTypes.bool.isRequired,
+        isChecked: PropTypes.bool.isRequired,
+        isExpanded: PropTypes.bool.isRequired,
+        isChildrenLoading: PropTypes.bool.isRequired,
+        type: PropTypes.string.isRequired,
+        data: PropTypes.shape({
+          title: PropTypes.string.isRequired,
+        }).isRequired,
+        defaults: PropTypes.shape({
+          isChecked: PropTypes.bool,
+          isExpanded: PropTypes.bool,
+        }),
+      }),
+    ).isRequired,
+  }),
+
+  /**
+   * CSS class to apply on each item.
+   */
+  classNameItem: PropTypes.string,
```

Comment:
For consistency (see e.g. Button.js) please use `className`

Reply:
`className`, it  feels  it's for the parent component.  but it's for the item.